package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "api-server",
+  "version": "1.0.0",
+  "description": "## Author: Lulu Sevignon",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Hey Lulu! This is causing some errors, not allowing tests to run. If you could change it to the following, that would be great!
```
"test": "jest --verbose --coverage",
"test-watch": "jest --watchAll --verbose --coverage",
```